src/Config/PublishMosaicConfig.ts

```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Directory from '../Directory';
+
+/**
+ * Publish Mosaic Config.
+ */
+export default class PublishMosaicConfig {
+
+  /**
+   * copy over all the non existent files to publish folder
+   */
+  public static publish(): void {
+    const publishMosaicConfigDir = Directory.getPublishMosaicConfigDir;
+    fs.ensureDirSync(publishMosaicConfigDir);
+    const publishedFileNames: Array<string> = fs.readdirSync(publishMosaicConfigDir);
+    const publishedFileNamesMap = {};
+    for (const publishedFileName of publishedFileNames) {
```

Comment:
May be?
```
    publishedFileNames.forEach((publishedFileName) => {
      publishedFileNamesMap[publishedFileName] = 1;
    });
```

Reply:
Not sure about the advantages of `forEach` over the above. 

Is the reason behind suggesting this more as code style thing? To promote all devs using same method for the same need ?

Reply:
Yes, Probably. I am also not sure which is more better and why. This decision is also not in the scope of this PR. We can visit this later.